eclide/ChildBuilderFrame.cpp

```diff
@@ -1124,6 +1124,26 @@ HWND OpenBuilderMDI(CMainFrame* pFrame, IAttribute *src, IWorkspaceItem * worksp
     return ((CMDIChildWnd *)pChild)->GetSafeHwnd();
 }
 
+bool OpenFileBuilderMDI(CMainFrame* pFrame, const CString & filePath, IWorkspaceItem * workspaceItem, bool locked, int row, int col, int len)
+{
+    CChildBuilderFrm* pChild = NULL;
+    if (!RestoreExisting(workspaceItem, pChild))
+    {
+        ATLASSERT(!filePath.IsEmpty());
+        pChild = new CChildBuilderFrm(workspaceItem);
+        CreateNewChild(pFrame, pChild, IDR_BUILDERWINDOW, workspaceItem->GetLabel());
+        if (!pChild->m_view->DoFileOpen(filePath))
+        {
+            pChild->DestroyWindow();
+            return false;
+        }
+        pChild->m_view->SetReadOnly(locked);
+    }
+    unsigned int pos = pChild->m_view->m_dlgview.m_view.PositionFromLine(row - 1) + col - 1;
+    pChild->m_view->m_dlgview.m_view.SetSel(pos, pos + len);
```

Comment:
Add a method to SourceView.h ```SetSel``` and then internally pass the call onto m_dlgView (IOW pretend m_dlgView is private/protected.